packages/app-builder/src/routes/ressources+/cases+/get-ai-settings.tsx

```diff
@@ -0,0 +1,14 @@
+import { initServerServices } from '@app-builder/services/init.server';
+import { getRoute } from '@app-builder/utils/routes';
+import { type LoaderFunctionArgs } from '@remix-run/node';
+
+export async function loader({ request }: LoaderFunctionArgs) {
+  const { authService } = initServerServices(request);
+  const { aiAssistSettings } = await authService.isAuthenticated(request, {
+    failureRedirect: getRoute('/sign-in'),
+  });
+
+  const settings = await aiAssistSettings.getAiAssistSettings();
+
+  return Response.json({ settings });
+}
```

Comment:
Can you use the middleware system? There's simple example in this PR (like: `/ressources+/analytics+/case-status-by-inbox.tsx`.